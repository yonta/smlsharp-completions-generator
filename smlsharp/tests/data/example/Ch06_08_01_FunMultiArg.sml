fun powerUncurry (0,C) = 1
  | powerUncurry (n,C) = C * powerUncurry(n - 1, C);
powerUncurry (2, 3);
fun powerCurry 0 C= 1
  | powerCurry n C = C * (powerCurry (n - 1) C);
powerCurry 2 3;

