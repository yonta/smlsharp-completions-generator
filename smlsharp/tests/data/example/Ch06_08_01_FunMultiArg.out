val powerUncurry = fn : int * int -> int

val it = 9 : int

val powerCurry = fn : int -> int -> int

val it = 9 : int
